README.md

```diff
@@ -1,15 +1,15 @@
-# OneTable
+# XTable
```

Comment:
According to the [branding guide](https://incubator.apache.org/guides/branding.html) and [ASF branding policy](https://www.apache.org/foundation/marks/pmcs) we must refer to the project as Apache XTable not simply as XTable.

The first most prominent reference to the project in every page/doc/etc must be Apache XTable for the rest it may be fine to leave it as XTable.

Reply:
+1  should it also say "(Incubating)" .

Reply:
Great catch, thank you! I updated this instance and the first instance in the docs readme and on the docs site as well.